### PR TITLE
Export 2019 01 08

### PR DESCRIPTION
**Pull request by benjamin-cheng**

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -27,6 +27,10 @@ msgctxt "action_delete"
 msgid "Delete"
 msgstr ""
 
+msgctxt "action_share"
+msgid "Share"
```
Comment (Delphine) [inline locales/templates/app.pot:33]: I think we'll need a quick localization note here to explain what is being shared

File: locales/templates/app.pot [hunk 2]
```diff
@@ -180,6 +184,25 @@ msgctxt "detail_ocr_error_notext"
...
 msgstr ""
 
+#. String displayed to ask users to connect to internet to install necessary
+#. module
+msgctxt "detail_ocr_error_module"
+msgid "Connect to the internet to install offline Text Recognition module"
+msgstr ""
+
+msgctxt "detail_ocr_error_action_connect"
+msgid "Connect"
```
Comment (Delphine) [inline locales/templates/app.pot:197]: I think we'll need a quick localization note here to explain what "connect" refers to exactly

File: locales/templates/app.pot [hunk 3]
```diff
@@ -579,3 +598,42 @@ msgctxt "onboarding_search_content_feedback"
...
+msgstr ""
+
+#. String displayed to give examples of what the new feature can do
+msgctxt "onboarding_ocr_content_examples"
+msgid "Look up products, copy coupon codes, and share quotes in your screenshots"
+msgstr ""
+
+#. In-app promote dialog
+msgctxt "dialogue_feedback_title"
+msgid "Love Firefox ScreenshotGo?"
```
Comment (Delphine) [inline locales/templates/app.pot]: %1$s has been used for "Firefox ScreenshotGo", probably better to be consistent with that

File: locales/templates/app.pot [hunk 4]
```diff
@@ -579,3 +598,42 @@ msgctxt "onboarding_search_content_feedback"
...
+msgctxt "dialogue_feedback_action_5stars"
+msgid "Rate 5 Stars"
+msgstr ""
+
+msgctxt "dialogue_feedback_action_send"
+msgid "Send Feedback"
+msgstr ""
+
+msgctxt "dialogue_share_title"
+msgid "Share Firefox ScreenshotGo with Friends!"
```
Comment (Delphine) [inline locales/templates/app.pot]: %1$s has been used for "Firefox ScreenshotGo", probably better to be consistent with that

**Review by Delphine — CHANGES_REQUESTED** [review]
Just a few comments.
Additionally: it seems like there already are very similar strings that exist to dialog_feedback and share ones. Is this expected that these are added, and the former ones not removed?
(for example I see: settings_list_share "share with friends" in current strings)

File: locales/templates/app.pot (hunk 1 above)
Comment (ChenMorpheus) [reply locales/templates/app.pot:33]: Thanks for the suggestion. Done!

"Label used for buttons to share selected items, e.g. selected text, selected screenshot, or share the app"

File: locales/templates/app.pot (hunk 2 above)
Comment (ChenMorpheus) [reply locales/templates/app.pot:197]: Thanks for the suggestion. Done!

"String displayed to direct users to Network & internet in system Settings"

**Comment by ChenMorpheus** [comment]
@Delphine 

> Just a few comments.
Additionally: it seems like there already are very similar strings that exist to dialog_feedback and share ones. Is this expected that these are added, and the former ones not removed?
(for example I see: settings_list_share "share with friends" in current strings)

Except for the two strings I replaced with the previous copy, I didn't change the other strings considering the context for users. For example, "share with friends" in settings provides enough more context. As for the "share," it follows "Share Firefox ScreenshotGo with Friends," so a verb is sufficient.

**Comment by benjamin-cheng** [comment]
Hi @Delphine, we've updated the files to address your comments, could you please help to review? Thank you!

File: locales/templates/app.pot (hunk 1 above)
Comment (Delphine) [reply locales/templates/app.pot:33]: awesome, thank you!

File: locales/templates/app.pot (hunk 2 above)
Comment (Delphine) [reply locales/templates/app.pot:197]: thanks again! :)

**Review by Delphine — APPROVED** [review]
Thanks for the updates! LGTM now :)

**Pull request merged**